robolectric-annotations/build.gradle

```diff
@@ -0,0 +1,5 @@
+dependencies {
+    // Compile dependencies
+    provided "com.intellij:annotations:12.0"
+    provided "org.robolectric:android-all:6.0.0_r1-robolectric-0"
```

Comment:
Gotcha. I made a task for it in tracker. https://www.pivotaltracker.com/story/show/124305721